### DISCUSSION TITLE
Create a user account using a wallet

### DISCUSSION THREAD

**Discussion by shealan** [m1guelpf/laravel-web3-login#4] [Ideas]
Thanks for creating this package. It's good to see some web3 love in the Laravel community.

It would be great if you could create a new user account by simply connecting your wallet. Rather than creating the account first then assigning the eth wallet access.

**Comment by sawirricardo** [comment]
@shealan Hm I see that this package currently not support that yet. However, if I want to have a fresh start, here's what I'm going to do:
1. Edit User migration, to make email, name, password nullable.
2. When user connect wallet, simply check if user with the address 0x1xxxxxx exists
3. if not exist, just create user.

Do you have any other idea? or do I miss something? Let me know

**Comment by yudikubota** [comment]
I had the same issue so I made a PR https://github.com/m1guelpf/laravel-web3-login/pull/10 with the changes I made to make it work.
Maybe this works for you too.
